src/components/cards/RestTimerSetup.js

```diff
@@ -0,0 +1,39 @@
+
+import React from 'react';
+
+import Button from '../bites/Button';
+
+import CronometerTimeView from '../../utils/CronometerTimeView';
+
+function RestTimerSetup({ restTime, setRestTimeButton }) {
+  return (
+    <div className="RestTime">
+      <h3 className="text-3xl font-bold p-2">Rest Time</h3>
+      <div className="flex justify-center items-center">
+        <div className="flex flex-col col-25">
+          <Button onClickFn={setRestTimeButton} onClickParams={-1} text="-0.1s" color="red" />
```

Comment:
This multiples buttons declarations, could be an array, coverted 'onClickParams' and 'text' to object params

Reply:
Updated in 834ebe8